{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"causedBy"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["macOS"]}]},{"kind":"parameters","parameters":[{"name":"Error","content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"The original error that cause the I\/O error."}]}]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/iostreams\/ioerror\/causedby(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/IOStreams\/documentation\/IOStreams\/IOError\/causedBy(_:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"A non-specific I\/O errorr occurred that was caused by another error."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"causedBy"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}],"title":"IOError.causedBy(_:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:9IOStreams7IOErrorO8causedByyACs5Error_pcACmF","modules":[{"name":"IOStreams"}]},"hierarchy":{"paths":[["doc:\/\/IOStreams\/documentation\/IOStreams","doc:\/\/IOStreams\/documentation\/IOStreams\/IOError"]]},"references":{"doc://IOStreams/documentation/IOStreams":{"role":"collection","title":"IOStreams","abstract":[],"identifier":"doc:\/\/IOStreams\/documentation\/IOStreams","kind":"symbol","type":"topic","url":"\/documentation\/iostreams"},"doc://IOStreams/documentation/IOStreams/IOError":{"role":"symbol","title":"IOError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"IOError"}],"abstract":[{"type":"text","text":"I\/O related errors"}],"identifier":"doc:\/\/IOStreams\/documentation\/IOStreams\/IOError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"IOError"}],"url":"\/documentation\/iostreams\/ioerror"},"doc://IOStreams/documentation/IOStreams/IOError/causedBy(_:)":{"role":"symbol","title":"IOError.causedBy(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"causedBy"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}],"abstract":[{"type":"text","text":"A non-specific I\/O errorr occurred that was caused by another error."}],"identifier":"doc:\/\/IOStreams\/documentation\/IOStreams\/IOError\/causedBy(_:)","kind":"symbol","type":"topic","url":"\/documentation\/iostreams\/ioerror\/causedby(_:)"}}}